content/_releases/22/46.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Product Update - v22.46
+description: Details of the product update v22.46 from 17th of November 2022.
+layout: rel
+version: "22.46"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-11-17
+redirect_from:
+  - /releases/22.46.html
+---
+
+## Improvements
+
+### Mapper can map objects
```

Comment:
Mapper can now make objects in heirarchies (otherwise it sounds more like a bugfix)